guides/v2.0/get-started/soap/soap-web-api-calls.md

```diff
@@ -88,7 +88,7 @@ $wsdlUrl = 'http://magento.ll/soap/default?wsdl=1&services=testModule1AllSoapAnd
 $serviceArgs = array("id"=>1);
 
 $context = stream_context_create($opts);
-$soapClient = new SoapClient($wsdlUrl, ['version' => SOAP_1_2, 'context' => $context]);
+$soapClient = new SoapClient($wsdlUrl, ['version' => SOAP_1_2, 'stream_context' => $context]);
```

Comment:
Technically, we no longer support 2.0 docs since Magento no longer supports v2.0. 

To make sure this update propagates to the more current documentation (2.1, 2.2, 2.3), you must update the 2.1 version of the file instead: 'guides/v2.0/get-started/soap/soap-web-api-calls.md'. 2.2 is symlinked to 2.1 and 2.3 is symlinked to 2.2.

Reply:
It appears that this applies to 2.0 only. In this case, I think it's valid to update the 2.0 docs, especially since someone reported a doc issue about it and we don't want to keep publishing bad info. @erikmarr, please go ahead and process.